fontheight-core/src/exemplars.rs

```diff
@@ -75,6 +103,8 @@ impl<'w> ExemplarCollector<'w> {
             if by_lowest < *weakest_low {
                 *weakest_low = by_lowest;
             }
+        } else {
+            panic!("pushing to zero capacity ExemplarCollector");
```

Comment:
nit: IMO this is a little panic-heavy; if we really don't want it to happen then we should panic earlier in the constructor, but I'm not sure if preventing a zero-length container from being constructed is too important